src/platform/tls_schannel.c

```diff
@@ -675,6 +695,37 @@ QuicTlsLibraryInitialize(
         goto Error;
     }
 
+    Status =
```

Comment:
The code below needs to be resilent to being run on older OSes. So we can't fail initializing the library if ChaCha/Poly isn't available